lfsapi/client.go

```diff
@@ -23,20 +23,37 @@ import (
 const MediaType = "application/vnd.git-lfs+json; charset=utf-8"
 
 var (
-	UserAgent = "git-lfs"
-	httpRE    = regexp.MustCompile(`\Ahttps?://`)
+	UserAgent                 = "git-lfs"
+	httpRE                    = regexp.MustCompile(`\Ahttps?://`)
+	maxSshAuthenticateRetries = 5
```

Comment:
Do you think that this could be an instance variable belonging to `*lfsapi.Client`? I don't think that this has to be in the package-global namespace, so I'd advocate for the former.